File: SistemaProcinco/SistemaProcinco.Common/Models/PantallasPorRolesViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SistemaProcinco.Common.Models
{
    public class PantallasPorRolesViewModel
    {
        public int PaPr_Id { get; set; }
        public int Pant_Id { get; set; }
        [NotMapped]
        public string Pantalla { get; set; }
        public int Role_Id { get; set; }
        [NotMapped]
        public string Rol { get; set; }
        public int? PaPr_UsuarioCreacion { get; set; }
        public DateTime? PaPr_FechaCreacion { get; set; }
        public int? PaPr_UsuarioModificacion { get; set; }
        public DateTime? PaPr_FechaModificacion { get; set; }
        public bool? PaPr_Estado { get; set; }

        [NotMapped]
        public string Creacion { get; set; }

        [NotMapped]
        public string Modificacion { get; set; }
    }
}

File: PlantillaProcinco/src/app/Models/PantallasPorRolesViewModel.ts

export class PantallasView {
    pant_Id !: Number;
    pant_Descripcion  !: String;
    pant_UsuarioCreacion !: Number;
    pant_FechaCreacion !: String;
    pant_UsuarioModificacion !: Number;
    pant_FechaModificacion !: String;
    pant_Estado !: boolean;
    creacion !: String;
    modificacion !: String;
}


export class PantallasPorRolesView {
    paPr_Id !: Number;
    pant_Id !: Number;
    paPr_UsuarioCreacion !: Number;
    paPr_FechaCreacion !: String;
    paPr_UsuarioModificacion !: Number;
    paPr_FechaModificacion !: String;
    paPr_Estado !: boolean;
    creacion !: String;
    modificacion !: String;
}
